src/extension.ts

```diff
@@ -954,6 +954,31 @@ export function getUSSProfile(node: ZoweUSSNode) {
     return profile;
 }
 
+function appendSuffix(label: string): string {
```

Comment:
I can see this being a problem if someone has a dataset named `JASMINE.APPLIED.COBOL.SCRIPTS`
Maybe I'm trying to determine if there is a way in which we can limit this to one suffix only. Maybe go from LLQ to HLQ (i.e. backwards) trying to match an expected suffix.

Reply:
I agree, it's never going to be an exact science and there isn't a way to override in this situation. I did toy with the idea of using the LLQ itself as the suffix and that way it would be down to the individual using VSCode how they would map that suffix to a syntax. For example mapping JASMINE.APPLIED.COBOL(member).**COBOL** the same as if it read .cbl

Maybe the compromise is to put the LLQ as the suffix as well (in lower case) but recognize certain ones like COBOL and replace with the suffix of .CBL like this
JASMINE.APPLIED.COBOL(member).**cbl**

Reply:
Going back to your example. If I handle a default approach as 
`JASMINE.APPLIED.COBOL.AMISC(member)`
gets saved as 
`JASMINE.APPLIED.COBOL.AMISC(member).amisc`

your example above becomes
`JASMINE.APPLIED.COBOL.SCRIPTS(member)`
gets saved as 
`JASMINE.APPLIED.COBOL.SCRIPTS(member).sh`

and for shorter
`JASMINE.APPLIED.COBOL(member)`
gets saved as 
`JASMINE.APPLIED.COBOL(member).cbl`

That way as all items have a suffix users would always be able to associate with a syntax highlighter

Reply:
The only question is do we still traverse backwards to find an overriding suffix in the first place

Reply:
Okay implemented as above. Working backwards as priority.